app/src/styles/globals.css

```diff
@@ -24,73 +24,73 @@
 
```

Comment:
Moved color types like `hsl` and `rgb` from tailwind.config.js to here. This seemed to be necessary for the sake of having variables override / fall back to each other; it's also nicer to have the color channels defined next to the color types.